projects/plugins/jetpack/_inc/lib/core-api/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-external-media.php

```diff
@@ -48,6 +48,21 @@ class WPCOM_REST_API_V2_Endpoint_External_Media extends WP_REST_Controller {
 			'title'   => array(
 				'type' => 'string',
 			),
+			'meta'    => array(
+				'type'                 => 'object',
+				'additionalProperties' => false,
+				'properties'           => array(
+					'vertical_id'   => array(
+						'type' => 'string',
+					),
+					'pexels_object' => array(
+						'type' => 'object',
+					),
+					'orientations'  => array(
+						'type' => 'array',
```

Comment:
I'm not sure if defining [an enum parameter](https://github.com/WordPress/wordpress-develop/blob/b64d06f7840a18f86a42ad14c8bff80fe6fc1456/src/wp-includes/rest-api/endpoints/class-wp-rest-posts-controller.php#L2442) would help with validation here?

Reply:
I added this, but it doesn't actually seem to work. Probably because of the same nested level thing. Maybe I should keep it anyway just to be more explicit?

Reply:
Ah that is really too bad